### PR TITLE
Table serializable

### PR DESCRIPTION
**Pull request by robinborst95**
Tables are now serializable and can be written to a file.
So, in the visualization the original table can be retrieved by reading the corresponding file.
Also added a getColumns method to Table.

### CODE REVIEW

**Comment by paulbakker65** [comment]
Looks great! Everything passed and this makes visualization a lot easier!

File: src/main/java/main/MainUI.java [hunk 1]
```diff
@@ -315,16 +321,16 @@ public static File openSettingsFile() {
...
       }
-      System.out.println("Selected settings file: " + file);
+      System.out.println("Selected " + title + ": " + file);
       return file;
     }
-
+    
     return null;
-  }
+  }
```
Comment (grootstyr) [inline src/main/java/main/MainUI.java]: Whitespace errors

**Comment by grootstyr** [comment]
Formatting isn't checkstyle compliant again.

**Comment by Unset** [comment]
Ok seems good

**Comment by jthols** [comment]
Looks good!

**Pull request merged**